test/unit/stringify.test.js

```diff
@@ -1,40 +1,48 @@
 import assert from 'assert';
-import stringify from '../../src/stringify.js';
+import { unsafeStringify, stringify } from '../../src/stringify.js';
```

Comment:
With the above change, I think this changes as follows (right?)
```suggestion
import stringify, { unsafeStringify } from '../../src/stringify.js';
```